### DISCUSSION TITLE
Wrong device found by serial number?

### DISCUSSION THREAD

**Discussion by petjek** [tbnobody/OpenDTU#2020] [General]
Hi there,
I added the device serial number just to find out it shows a HM-600/700/800-2T. But the device actually is a HMS-2000-4T. Only 2 strings show up and no data is transmitted. Even no information available in console view. There is a thread here saying v24.1.26 works. Gave it a try but no change.

**Comment by ms49434** [comment]
> Hi there, I added the device serial number just to find out it shows a HM-600/700/800-2T. But the device actually is a HMS-2000-4T. Only 2 strings show up and no data is transmitted. Even no information available in console view. There is a thread here saying v24.1.26 works. Gave it a try but no change.

A HMS-2000-4T is identified by the first four digits of its serial number which is 1164. What does your serial number look like?

**Comment by petjek** [comment]
Exactly like that: 1146Axxxxxxx

**Reply by ms49434** [reply]
1146 is not 1164..., possibly a typo?

**Reply by petjek** [reply]
OMG, I'm such an idiot. Thx, works like a charm now.

**Comment by MetaChuh** [comment]
@petjek

many thanks for opening your topic in the discussions area, instead of opening an issue.
much appreciated.

please mark @ms49434 's answer as solution.
marking solutions helps users with similar questions to solve their current needs.

thx & greetings,
metachuh
